woocommerce/woocommerce-admin
# Add Google Ads & Marketing by Kliken Note Provider

See https://github.com/Automattic/woocommerce.com/issues/8471

Replace the  Facebook Marketing Experts offer with a note for Google Ads & Marketing by Kliken.

Note should be displayed if Google Ads & Marketing by Kliken is installed and the store has at least 20 orders in the past month.

I'm still a little fuzzy on when we should be deleting old notes - I've opted to remove the facebook note on update in this PR similar to what was done previously here https://github.com/woocommerce/woocommerce-admin/pull/4030 - happy for thoughts/feedback/clarification on when to or not to do that. 

In the future this sort of note would be probably be a good candidate for the remote inbox notifications.

### Screenshots

![Screen Shot 2020-09-09 at 10 21 32 am](https://user-images.githubusercontent.com/355014/92541865-53947880-f286-11ea-8e92-ab7b9d225ae0.png)

### Detailed test instructions:

- Create 20 tests orders on your test site in the past month (if you don't already have that many) or adjust `possibly_add_note()` to check for a lower number.
- Install Google Ads & Marketing by Kliken
- Run `wc_admin_daily` cron event (I use WP Crontrol) just to ensure the note event is run.
- Check you have a new inbox note about Google Ads & Marketing by Kliken.
- Check the action links to the WCCOM product page

### Changelog Note:

Enhancement: Replace Facebook Marketing Expert Offer Inbox Note with Google Ads & Marketing Inbox Note
